GVFS/GVFS.Common/Git/GitProcess.cs

```diff
@@ -108,6 +109,23 @@ public static bool TryGetVersion(string gitBinPath, out GitVersion gitVersion, o
             return true;
         }
 
+        public bool KillRunningProcess()
+        {
+            try
+            {
+                if (this.executingProcess != null)
+                {
+                    this.executingProcess.Kill();

```

Comment:
What will `Kill` do if the process has already been killed? Will it throw?

Reply:
It will! I'll add the necessary `catch` statements as specified in [the Process.Kill() docs](https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.process.kill?view=netframework-4.7.2).